site/overview/faqs.md

```diff
@@ -0,0 +1,154 @@
+
+::: details Answer
+We front the gas for your application and put the USD equivalent on your bill at the end of the month. No need to worry about pre-funding the gas manager or conversions, we’ve got you covered!
+:::
+
+### What are my gas sponsorship limits?
+
+::: details Answer
+You can find details of gas manager limits depending on your tier [here](https://docs.alchemy.com/reference/gas-manager-coverage-api-pricing#fee-logic).
+:::
+
+### Do you support ERC20 or stablecoin paymasters?
+
+::: details Answer
+Currently, we don’t support this, but we are actively exploring. Please [reach out](/overview/contact-us) if you are interested as we would love your input in our spec.
+:::
+
+### How is the gas manager protected from DDOS attacks?
+
+::: details Answer
+In your gas manager policy, you can configure spending rules per address, per app, and/or policy wide limits. See how to set up these policies [here](/tutorials/sponsoring-gas/sponsoring-gas.html#_2-create-a-gas-manager-policy).
+:::
+
+## Common Errors {#common-errors}
+
+### Invalid policy ID: `{ code: -32602, message: 'Invalid Policy ID' }`
+
+::: details Answer
+Gas manager policies can only be tied to one app. Make sure you are using the app key id that is associated with the app the Gas Manager policy is configured for or create a new policy for the app you are using.
+:::
+
+### Precheck failed: `{ code: -3200, message: 'precheck failed: ...' }`
+
+::: details Answer
+Precheck failed errors are often related to gas and/or fees. Our bundler follows standard [ERC 4337](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-4337.md#client-behavior-upon-receiving-a-useroperation) implementation for gas and fee checks in order to 1) ensure your user operations land on chain and to 2) protect the bundler from potential attacks in order to support scalability.
+
+These errors are often related to market movement between the time gas and fees are estimated and the time when user operations are submitted to the bundler. This fluctuation in the market is especially variant on testnet. We currently reject upon sending if the user operation is underpriced compared to the network rate to ensure your user operation is included in a block.
+
+To handle these errors, we recommend you use our [override fields](/packages/aa-core/provider/types/userOperationOverrides) to increase buffers on top of our estimates and implement retry mechanisms as needed.
+
+Our gas and fee estimations are just that, estimations, but we are always working to improve these estimates!
+:::
+
+### Total execution gas limit exceeded: `{ code: -3202, message: 'precheck failed: total execution gas is X but must be at most 10000000}`
+
+::: details Answer
+Currently our bundler allows max 10M gas in aggregate between preVerificationGas, verificationGasLimit, and callGasLimit. To reduce the gas needed, try reducing the size of your call data and/or sending your call data in multiple user operations rather than one.
```

Comment:
```suggestion
Currently our bundler allows max 10M gas in aggregate between `preVerificationGas`, `verificationGasLimit`, and `callGasLimit`. To reduce the gas needed, try reducing the size of your call data and/or sending your call data in multiple user operations rather than one.
```